common/src/main/kotlin/no/nav/su/se/bakover/common/periode/Periode.kt

```diff
@@ -36,6 +36,8 @@ data class Periode private constructor(
         return sluttStart == plussEnDag || sluttStart == minusEnDag || startSlutt == plussEnDag || startSlutt == minusEnDag
     }
 
+    infix fun overlapper(other: List<Periode>): Boolean = this.tilMånedsperioder().map{it.fraOgMed}.sorted() == other.flatMap { it.tilMånedsperioder().map{it.fraOgMed} }.sorted()
```

Comment:
Hvordan funker denne implementasjonen for f.eks caset:  
```
Periode.create(1.januar(2021), 31.desember(2021)) overlapper
            listOf(
                Periode.create(1.januar(2021), 31.desember(2021)),
                Periode.create(1.januar(2021), 31.desember(2021)),
                Periode.create(1.januar(2021), 31.desember(2021)),
            )
```
Med utgangspunkt i det funksjonelle caset vi forsøker å løse "inneholder periode a alle de samme månedene som periode n1..n" ville jeg vel forventet at dette ble sant? Kan vel sikkert løses ved å bruke et `Set`.

Må kanskje ta en liten runde på navnet også, litt forvirrende at overlapper ikke betyr det samme for en liste av perioder vs en enkelt periode. Kanskje noe i duren av `ekvivalentMed(other: List<Periode>)` kunne fungert? Andre forslag; kunne vi løst problemet generelt med mengdeoperasjoner? Tenker i baner av `snitt(a, b) == a` eller `komplement(a, b) == emtpySet` eller lignende?